Sources/RoutingKit/TrieRouter.swift

```diff
@@ -204,31 +204,29 @@ extension TrieRouter {
         
         var description: String {
             var desc: [String] = []
-            if let (name, parameter) = self.parameter {
+            for (name, constant) in self.constants {
                 desc.append("→ \(name)")
-                desc.append(parameter.description.indented())
+                desc += constant.description.indentSliced
             }
-            if let catchall = self.catchall {
-                desc.append("→ *")
-                desc.append(catchall.description.indented())
+            if let (name, parameter) = self.parameter {
+                desc.append("→ :\(name)")
+                desc += parameter.description.indentSliced
```

Comment:
Hmm, why change to `+=` here but continue using `append` on the previous line?

Reply:
I’ve changed `indented()` into `indentSliced`. Changing this into `+=` is because `indentSliced` has omitted the wrapping process in `indented()`. With this change, elements in `desc` will be “clearer” because they’re now all single lines. 

I propose to make it faster by setting a computed property that directly returns `[String]`, which can avoid all the unnecessary conversions here.

Reply:
I’ll push a new commit to implement that, in 12 hours since it’s late at night here where I am.